### PR TITLE
Weekly containers

### PR DESCRIPTION
**Pull request by HarshilPan07**
- added container labels to all hours of all days
- currently working on implementing nav js so it populates

### CODE REVIEW

File: source/js/weekly_navigation.js [hunk 1]
```diff
@@ -63,30 +180,117 @@ function populate(key) {
...
+        // loop through every log
         const tempKey = getDaysKey(window.curDate);
+        const log = getDaysData(tempKey);
+        const allTasks = log.tasks;
+        window.curDate.setDate(window.curDate.getDate() + 1);
+        
+        if(test_events[tempKey] == undefined){ 
+            continue;
+        }
+        console.log(tempKey + ", " + test_events[tempKey].name);
```
Comment (hexalellogram) [inline source/js/weekly_navigation.js:193]: Codacy has a fix for the issue: [Unexpected string concatenation.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7575454)
```suggestion
        console.log(`${tempKey  }, ${  test_events[tempKey].name}`);
```

File: source/js/weekly_navigation.js [hunk 2]
```diff
@@ -63,30 +180,117 @@ function populate(key) {
...
+                bottom = 10 + 10*(end_time_minutes/60);
+                bottom = '-' + bottom + '%';
+                event_label.style.bottom = bottom; 
+            } else {
+                bottom = 20 + 10*(10-end_time_hr) + 10*(end_time_minutes/60);
+                bottom = '-' + bottom + '%';
+                event_label.style.bottom = bottom;
+            }
+            
+            const id = 'cal-' + day.slice(0, 3).toLowerCase();
```
Comment (hexalellogram) [inline source/js/weekly_navigation.js:267]: Codacy has a fix for the issue: [Unexpected string concatenation.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7575454)
```suggestion
            const id = `cal-${  day.slice(0, 3).toLowerCase()}`;
```

File: source/js/weekly_navigation.js [hunk 3]
```diff
@@ -63,30 +180,117 @@ function populate(key) {
...
+                less than  10 and AM:
+            */
+            let bottom;
+            if(end_time_hr < 10 && end_time_ampm == 'AM') { 
+                bottom = 10*(10-end_time_hr) + 10*(end_time_minutes/60);
+                bottom += '%';
+                event_label.style.bottom = bottom; 
+            } else if(end_time_hr > 10 && end_time_ampm == 'PM') {
+                bottom = 10 + 10*(end_time_minutes/60);
+                bottom = '-' + bottom + '%';
```
Comment (hexalellogram) [inline source/js/weekly_navigation.js:259]: Codacy has a fix for the issue: [Unexpected string concatenation.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7575454)
```suggestion
                bottom = `-${  bottom  }%`;
```

File: source/js/weekly_navigation.js [hunk 4]
```diff
@@ -43,12 +43,129 @@ const sideBar = document.querySelector('#mySideBar ul');
 // the function content is at the bottom of the file
 let sideBarClick = function sideBarClickedTemp() {};
 
+
+ // Test Events
+ let test_events = {
```
Comment (hexalellogram) [inline source/js/weekly_navigation.js:48]: Codacy has a fix for the issue: ['test_events' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7575454)
```suggestion
    const test_events = {
```

File: source/js/weekly_navigation.js [hunk 5]
```diff
@@ -63,30 +73,129 @@ function populate(key) {
...
+       
         // eslint-disable-next-line prefer-destructuring
         days[i].innerText = getName(tempKey).split(',')[1];
-        window.curDate.setDate(window.curDate.getDate() + 1);
+    
+
+        const taskDay = log.name.slice(0, 3).toLowerCase();
+        const taskID = ['cal', 'task', taskDay].join('-');
+        const taskBox = document.getElementById(taskID);
+        const id = 'cal-' + day.slice(0, 3).toLowerCase();
```
Comment (hexalellogram) [inline source/js/weekly_navigation.js:92]: Codacy has a fix for the issue: [Unexpected string concatenation.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7575454)
```suggestion
            const id = `cal-${  day.slice(0, 3).toLowerCase()}`;
```

File: source/js/weekly_navigation.js [hunk 6]
```diff
@@ -63,30 +73,129 @@ function populate(key) {
...
+                less than  10 and AM:
+            */
+            let bottom;
+            if(end_time_hr < 10 && end_time_ampm == 'AM') { 
+                bottom = 10*(10-end_time_hr) - 10*(end_time_minutes/60);
+                bottom += '%';
+                event_label.style.bottom = bottom; 
+            } else if(end_time_hr >= 10 && end_time_ampm == 'AM') {
+                bottom = 10*end_time_hr%10 + 10*(end_time_minutes/60);
+                bottom = '-' + bottom + '%';
```
Comment (hexalellogram) [inline source/js/weekly_navigation.js:167]: Codacy has a fix for the issue: [Unexpected string concatenation.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7575454)
```suggestion
                bottom = `-${  bottom  }%`;
```

**Review by artyom2001 — APPROVED** [review]
Populates weekly log, looks good to me, so approved to merge!

**Pull request merged**